DRIQuery.cpp

```diff
@@ -160,7 +152,7 @@ bool DRIQuery::canHandle() {
         if (possibleExts.find("GLX_MESA_query_renderer") == std::string::npos ||
                 this->getScreenDriver == nullptr ||
                 this->getDriverConfig == nullptr) {
-            std::cerr << "Closed source driver!!" << std::endl;
+            std::cerr << _("Closed source driver!!") << std::endl;
```

Comment:
Can you please also update the translation files?
So that the po files have the new string.